### ISSUE TITLE
setup service architecture

### ISSUE THREAD

**Issue by flyingzumwalt** [projecthydra-labs/hydra-works#62]
[moved from projecthydra-labs/hydra-pcdm#57 by @elrayle] 
create the following directory structure in Hydra::Works
````
\lib\hydra\works\services
     \collection
     \generic_work
     \generic_file
     \file
````

**Comment by flyingzumwalt** [comment]
@elrayle can you provide some explanation of the purpose of these services?

**Comment by awead** [comment]
A lot of this was addressed in #105. Shall we close this @elrayle ?
